### ISSUE TITLE
Clean up the container registry

### ISSUE THREAD

**Issue by berezovskyi** [EricssonResearch/scott-eu#145]
Delete the `scott/scott-eu/scott` images, no longer used after #133

**Comment by berezovskyi** [comment]
NB https://git.cf.ericsson.net/scott/scott-eu/container_registry

**Comment by berezovskyi** [comment]
wontfix
